server_admin/topics/admin-console.adoc

```diff
@@ -7,12 +7,19 @@ include::realms/proc-using-admin-console.adoc[leveloffset=1]
 include::realms/master.adoc[leveloffset=2]
 include::realms/proc-creating-a-realm.adoc[leveloffset=2]
 include::realms/ssl.adoc[leveloffset=2][]
-include::realms/cache.adoc[leveloffset=2][]
```

Comment:
ifeval::[{project_product}==true]
include::realms/cache.adoc[leveloffset=2][]
endif::[]

Reply:
This section will still be needed for SSO releases that will use the old Admin Console until next year.

Reply:
Thanks, I have added it back in.